addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -409,6 +421,22 @@ odoo.define('website.tour.form_editor', function (require) {
             // We have to this that way because the input type = hidden.
             extra_trigger: 'form:has(input[name="email_to"][value="test@test.test"])',
         },
+        {
+            content: "Enter in edit mode again",
+            trigger: "a[data-action='edit']",
+        },
```

Comment:
Any reason to actually exit edit mode? It's expensive timewise to leave edit mode, and to enter it too, so here doing both twice really takes time for no reason (if I am not wrong?)

Reply:
I only did that to keep all the content of the test apart from what it was existing but sure, I will move the `addCustomField` of the special characters into the last edit mode :+1: